lib/src/migration_visitor.dart

```diff
@@ -101,24 +113,22 @@ abstract class MigrationVisitor extends RecursiveAstVisitor {
   void visitDependency(Uri dependency, FileSpan context) {
     var result = importCache.import(dependency, _importer, _currentUrl);
     if (result != null) {
+      // If [dependency] comes from a non-relative import, don't migrate it,
+      // because it's likely to be outside the user's repository and may even be
+      // authored by a different person.
+      //
+      // TODO(nweiz): Add a flag to override this behavior for load paths
+      // (#104).
+      if (result.item1 != _importer) return;
+
+      var oldImporter = _importer;
       _importer = result.item1;
       var stylesheet = result.item2;
       visitStylesheet(stylesheet);
-    } else {
-      handleMissingDependency(dependency, context);
+      _importer = oldImporter;
     }
   }
 
-  /// Adds the missing [dependency] within the stylesheet at [source] to the
-  /// missing dependency list to warn after migration completes.
-  ///
-  /// Migrators should override this if they want a different behavior.
-  @protected
-  void handleMissingDependency(Uri dependency, FileSpan context) {
-    _missingDependencies.putIfAbsent(
```

Comment:
Tests are failing because this needs to be inlined into `visitDependency`

Reply:
Done.